pkg/ccl/multiregionccl/regionliveness_test.go

```diff
@@ -168,5 +168,5 @@ func TestRegionLivenessProber(t *testing.T) {
 			}
 			return nil
 		})
-	})
+	}, time.Minute*2)
```

Comment:
instead of making the test take longer, is there a way to make the timeout shorter? is that bad for other reasons?